### PR TITLE
docs: typos and grammar

### PR DESCRIPTION
**Pull request by fredericAerts**

### CODE REVIEW

**Comment by mr-zwets** [comment]
Thanks for the PR! 🙏  

You let me know that you'll still be going over the SDK docs to check for typos and grammar.
We'll wait to merge the PR for your full check-up on the docs! 😄

**Comment by fredericAerts** [comment]
I've gone through SDK section and committed changes. Check-up on the docs done for my part.

File: website/docs/sdk/instantiation.md [hunk 1]
```diff
@@ -2,14 +2,14 @@
...
+Before interacting with a smart contract on the BCH network, the CashScript SDK needs to instantiate a `Contract` object. This is done by providing the contract's information and constructor arguments. After this instantiation, the CashScript SDK can interact with the BCH contract.
 
 :::note
 CashScript only offers a JavaScript SDK, but CashScript contracts can be integrated into other languages as well. Because there are no ready-to-use SDKs available for them, this is considered advanced usage.
 :::
 
 ## Contract class
-The `Contract` class is used to represent a CashScript contract in a JavaScript object. These objects can be used to retrieve information such as the contract's address and balance. They can be used to interact with the contract by calling the contract's functions.
+The `Contract` class is used to represent a CashScript contract in a JavaScript object. These objects can be used to retrieve information such as the contract's address and balance,. They can also be used to interact with the contract by calling the contract's functions.
```
Comment (rkalis) [inline website/docs/sdk/instantiation.md:12]: Small extra comma here.

**Review by rkalis — CHANGES_REQUESTED** [review]
Hey @fredericAerts , thanks so much for the contribution! There's just the one place with a `,.`, besides that everything looks great. If you could edit that one issue we'll get this merged in.

File: website/docs/sdk/instantiation.md (hunk 1 above)
Comment (fredericAerts) [reply website/docs/sdk/instantiation.md]: Nice that you noticed that one!

Comment (rkalis) [reply website/docs/sdk/instantiation.md]: Thanks for the quick fix!

**Review by rkalis — APPROVED** [review]

**Comment by rkalis** [comment]
Thank you again for the contribution!

**Pull request merged**